tests/simpleTest/simple_app_state.hpp

```diff
@@ -0,0 +1,139 @@
+// Concord
```

Comment:
move the implementation into a .cpp file and keep only definitions.

Reply:
Fixed